Documentation/ApiOverview/Fluid/Index.rst

```diff
@@ -109,23 +132,75 @@ you must define the sections that are referenced by the layout.
 :file:`Layouts`
 ---------------
 
+*optional*
+
 Layouts serve as a wrapper for a web page or a specific block of content. If using Fluid
 for a sitepackage, a single layout file will often contain multiple components such as your
 sites menu, footer, and any other items that are reused throughout your website.
 
+Templates can be used with or without a Layout.
+
+* *With a Layout* anything that's not inside a section is ignored. When a Layout is used,
+  the Layout determines which sections will be rendered from the template through use of
+  :xml:`<f:render>` in the Layout file.
+* *Without a Layout* anything that's not inside a section is rendered. You can still use
+  sections of course, but you then must use f:render in the template file itself, outside
+  of a section, to render a section.
+
 :file:`Partials`
 ----------------
 
-*Using Partials is optional.*
+*optional*
 
 Partials are a Fluid component. Partials can be used as reusable components from within
 a template.
 
-Example using Layout
---------------------
+Example
+-------
 
 This example was taken from the `example extension <https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage-Code/>`__
-for :ref:`t3sitepackage:start` and reduced to a very basic example:
+for :ref:`t3sitepackage:start` and reduced to a very basic example.
+
+The Sitepackage Tutorial walks you through the creation of a sitepackage (theme) using Fluid.
+In our simplified example here, the general structure of a page is defined by a layout "Default".
```

Comment:
In our example, the overall structure of a page is defined by a layout "Default".